example/src/main/kotlin/com.expedia.graphql.sample/query/UnionQuery.kt

```diff
@@ -0,0 +1,17 @@
+package com.expedia.graphql.sample.query
+
+import org.springframework.stereotype.Component
+
+@Component
+class QueryWithUnion: Query {
+    fun whichHand(whichHand: String): AUnionType = when(whichHand) {
+        "right" -> RightHand(12)
+        else -> LeftHand("hello world")
+    }
+}
+
+interface AUnionType
```

Comment:
by convention I put all model classes into `model` package, lets keep it consistent